packages/ai/src/ai/account/file_store.py

```
@@ -0,0 +1,425 @@
+            path: Relative path within the account store.
+            connection_id: ID of the owning connection (for cleanup on disconnect).
+
+        Returns:
+            Handle ID string.
+
+        Raises:
+            StorageError: If the path is already open for writing.
+        """
+        full_path = self._full_path(path)
+        if full_path in self._write_locks:
+            raise StorageError(f'File already open for writing: {path}')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)